sysutils/clamav-server/Portfile

```diff
@@ -3,115 +3,321 @@
 PortSystem              1.0
 
 name                    clamav-server
-version                 0.98.4
+version                 0.101.2
 homepage                http://www.clamav.net/
 categories              sysutils
 platforms               darwin
 supported_archs         noarch
-maintainers             nomaintainer
-license                 BSD
+maintainers             @essandess
```

Comment:
Two comments on this:
 - Normally we add an email here too. Are you adding that for some reason?
 - Can `openmaintainer` be added?

Reply:
Thanks. Done.